src/components/KMultiselect/KMultiselect.vue

```diff
@@ -604,11 +604,20 @@ const stageSelections = () => {
     if (elem) {
       const height = elem.clientHeight
       if (height > selectionsMaxHeight.value) {
-        const item = visibleSelectedItemsStaging.value.pop()
-        if (item && !invisibleSelectedItemsStagingSet.has(item.value)) {
-          invisibleSelectedItemsStagingSet.add(item.value)
-          invisibleSelectedItemsStaging.value.push(item)
-        }
+        // populate as much items as possible by testing the offsetTop
+        const $overflowedItems = Array.from(elem.querySelectorAll('.k-multiselect-selection-badge'))
```

Comment:
nit: we do not prefix variable names with `$` or any other prefix. Since there are two variants of the same variable name, it would be preferable to just name them differently. 

Can you please update all occurrences?